CinemaCat.Api/Models/PersonDetails.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json.Serialization;
+
+namespace CinemaCat.Api.Models
+{
+    public class PersonDetails
+    {
+        public Guid Id { get; init; }
+
+        public required Person Person { get; init; }
```

Comment:
not sure about embedding a document here to get only the only field it, but feel free to ignore the comment, since we are going to have more fields in Person